### ISSUE TITLE
train lora error

### ISSUE THREAD

**Issue by operationairstrike** [bmaltais/kohya_ss#1367]
[01:37:12] WARNING  The following values were not passed to `accelerate launch` and had defaults used      launch.py:890
                    instead:
                            `--num_processes` was set to a value of `0`
                            `--num_machines` was set to a value of `1`
                            `--mixed_precision` was set to a value of `'no'`
                            `--dynamo_backend` was set to a value of `'no'`
                    To avoid this warning pass in values for each of the problematic parameters or run
                    `accelerate config`.
WARNING[XFORMERS]: xFormers can't load C++/CUDA extensions. xFormers was built for:
    PyTorch 2.0.1+cu118 with CUDA 1108 (you have 2.0.1+cpu)
    Python  3.10.11 (you have 3.10.6)
  Please reinstall xformers (see https://github.com/facebookresearch/xformers#installing-xformers)
  Memory-efficient attention, SwiGLU, sparse and more won't be available.
  Set XFORMERS_MORE_DETAILS=1 for more details
prepare tokenizers
Downloading (…)olve/main/vocab.json: 100%|██████████████████████████████████████████| 862k/862k [00:00<00:00, 2.32MB/s]
C:\Users\janis\kohya_ss\venv\lib\site-packages\huggingface_hub\file_download.py:133: UserWarning: `huggingface_hub` cache-system uses symlinks by default to efficiently store duplicated files but your machine does not support them in C:\Users\janis\.cache\huggingface\hub. Caching files will still work but in a degraded version that might require more space on your disk. This warning can be disabled by setting the `HF_HUB_DISABLE_SYMLINKS_WARNING` environment variable. For more details, see https://huggingface.co/docs/huggingface_hub/how-to-cache#limitations.
To support symlinks on Windows, you either need to activate Developer Mode or to run Python as an administrator. In order to see activate developer mode, see this article: https://docs.microsoft.com/en-us/windows/apps/get-started/enable-your-device-for-development
  warnings.warn(message)
Downloading (…)olve/main/merges.txt: 100%|██████████████████████████████████████████| 525k/525k [00:00<00:00, 17.4MB/s]
Downloading (…)cial_tokens_map.json: 100%|█████████████████████████████████████████████| 389/389 [00:00<00:00, 766kB/s]
Downloading (…)okenizer_config.json: 100%|████████████████████████████████████████████████████| 904/904 [00:00<?, ?B/s]
Using DreamBooth method.
prepare images.
0 train images with repeating.
0 reg images.
no regularization images / 正則化画像が見つかりませんでした
[Dataset 0]
  batch_size: 5
  resolution: (1024, 1024)
  enable_bucket: False

[Dataset 0]
loading image sizes.
0it [00:00, ?it/s]
make buckets
number of images (including repeats) / 各bucketの画像枚数（繰り返し回数を含む）
C:\Users\janis\kohya_ss\venv\lib\site-packages\numpy\core\fromnumeric.py:3432: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
C:\Users\janis\kohya_ss\venv\lib\site-packages\numpy\core\_methods.py:190: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
mean ar error (without repeats): nan
No data found. Please verify arguments (train_data_dir must be the parent of folders with images) / 画像がありません。引数指定を確認してください（train_data_dirには画像があるフォルダではなく、画像があるフォルダの親フォルダを指定する必要があります
![Screenshot 2023-08-10 014333](https://github.com/bmaltais/kohya_ss/assets/127621147/a62f44dd-fea1-4394-a81e-fe5c65a89f74)
![Screenshot 2023-08-10 014356](https://github.com/bmaltais/kohya_ss/assets/127621147/32ddded4-b758-4303-b5ac-71c55c310bbe)

**Comment by kieranxsomer** [comment]
OH that's LIKELY that you don't have your images folder set up according to the directory structure - took me about 5 times to figure that out.

Under "IMAGES" you'll want your # of repeats and your concept as your folder

kinda like 5_illustration or something like that

**Comment by operationairstrike** [comment]
Ok

**Comment by friendlyFriend4000** [comment]
> 

thank you so much, i was losing my mind already and it's just a simple naming error.

**Comment by VladAndronik** [comment]
@kieranxsomer can you please elaborate more on this with an example
